packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/LeftMenuLinkContent.js

```diff
@@ -53,6 +56,12 @@ const LeftMenuLinkContent = ({
       <LinkLabel>{labelId}</LinkLabel>
     );
 
+  const handleClick = () => {
```

Comment:
Can you set it in a `useEffect` hook directly in the `<MarketPlacePage />` ?

Reply:
ok yes I considered it but wasn't sure which approach was better.  Thanks.